### ISSUE TITLE
deduper.blocker() crashes

### ISSUE THREAD

**Issue by Leighroyus** [dedupeio/dedupe#773]
Done my best to debug this but no luck. I'm trying to get dedupe to work with a MS-SQL connection on Windows (8). First I got the csv example to work, I then started moving through the pgsql_big_dedupe  example code and I got stuck at:

#### generator that yields unique `(block_key, donor_id)` tuples.

`print('writing blocking map')`

`c3 = con.cursor('donor_select2')`
`c3.execute(DONOR_SELECT)`
`full_data = ((row['donor_id'], row) for row in c3)`
`b_data = deduper.blocker(full_data)`

and it crashes out after the line:
`csv_writer.writerows(b_data)`

With the following error:
`Traceback (most recent call last):
  File "C:/PythonV/dedupeio/dedupe-examples/pgsql_big_dedupe_example/csa_big_dedupe_example.py", line 237, in <module>
    csv_writer.writerows(b_data)
  File "C:\PythonV\dedupeio\dedupe\lib\site-packages\dedupe\blocking.py", line 43, in __call__
    record_id, instance = record
TypeError: cannot unpack non-iterable int object`

I had the thought to test deduper.blocker(full_data) call on the csv example (which I had success running) but I still get the same error.

Any thoughts?

Cheers Leigh

**Comment by Leighroyus** [comment]
Done my best to debug this but no luck. I'm trying to get dedupe to work with a MS-SQL connection on Windows (8). First I got the csv example to work, I then started moving through the pgsql_big_dedupe example code and I got stuck at:

Now we are ready to write our blocking map table by creating a generator that yields unique (block_key, donor_id) tuples.

`print('writing blocking map')`
`c3 = con.cursor('donor_select2')`
`c3.execute(DONOR_SELECT)`
`full_data = ((row['donor_id'], row) for row in c3)`
`b_data = deduper.blocker(full_data)`

and it crashes out after the line:
`csv_writer.writerows(b_data)`

With the following error:
`Traceback (most recent call last):
File "C:/PythonV/dedupeio/dedupe-examples/pgsql_big_dedupe_example/csa_big_dedupe_example.py", line 237, in 
csv_writer.writerows(b_data)
File "C:\PythonV\dedupeio\dedupe\lib\site-packages\dedupe\blocking.py", line 43, in call
record_id, instance = record
TypeError: cannot unpack non-iterable int object`

I had the thought to test deduper.blocker(full_data) call on the csv example that worked but I still get the same error.

Any thoughts?

Cheers Leigh
